cmd/workload-controller-manager/app/options/options.go

```diff
@@ -14,19 +15,53 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package options provides the flags used for the controller manager.
+//
 package options
 
 import (
+	"fmt"
+	kubectrlmgrconfigv1alpha1 "k8s.io/kube-controller-manager/config/v1alpha1"
```

Comment:
Please copy necessary kube controller manager code to workload controller manager. Do not refer to kube controller manager code. Otherwise, it will be impossible to retire kube controller manager later.